migration-from-4x.md

```diff
@@ -286,7 +286,7 @@ editor.ui.registry.addButton('myButton', {
 
 `onpostrender` has been removed and replaced with `onSetup` for menu and toolbar components. There are 3 major changes:
 
-* While [`onpostrender`](https://www.tiny.cloud/docs/advanced/creating-a-custom-button/#togglebutton) only ran once, when the editor was created, [`onSetup`]({{site.baseurl}}/ui-components/typesoftoolbarbuttons/#basicbuttonexampleandexplanation) runs every time a component is rendered, e.g. for a menu item, every time its menu becomes visible.
+* While [`onpostrender`]({{site.baseurl}}/advanced/creating-a-custom-button/#togglebutton) only ran once, when the editor was created, [`onSetup`]({{site.baseurl}}/ui-components/typesoftoolbarbuttons/#basicbuttonexampleandexplanation) runs every time a component is rendered, e.g. for a menu item, every time its menu becomes visible.
```

Comment:
This looks like it should be a v4 link instead of a v5 link, as `onpostrender` was removed in v5.